concordium-consensus/src/Concordium/GlobalState/BlockState.hs

```diff
@@ -1509,6 +1509,11 @@ class (BlockStateQuery m) => BlockStateOperations m where
     -- | Get whether a protocol update is effective
     bsoIsProtocolUpdateEffective :: UpdatableBlockState m -> m Bool
 
+    -- | Update the count of missed rounds for given validators by the given
+    --  delta. Rounds are `missed` by a validator, if it has been elected leader but the
+    --  round did timeout.
+    bsoUpdateMissedRounds :: (PVSupportsDelegation (MPV m)) => UpdatableBlockState m -> [(BakerId, Word64)] -> m (UpdatableBlockState m)
```

Comment:
I think this should be constrained with `PVSupportsValidatorSuspension (MPV m)`. Otherwise, the implementation should actively check whether suspension is supported, and do nothing if it's not.

Reply:
The implementation does nothing if suspension is not supported, because the field is simply not there in this case. We can still constrain it here as well though, to make this a bit clearer.